packages/tofu/src/inspectSource.js

```diff
@@ -183,37 +183,110 @@ function inspectGlobals(
  * @param {import('@babel/types').Node} ast
  * @param {string[]} [packagesToInspect] - List of module IDs to look for; if
  *   omitted, will inspect all imports
+ * @param {boolean} [deep] Return named exports and imports
  * @returns {string[]} - List of imported identifiers
  */
-function inspectEsmImports(ast, packagesToInspect) {
+function inspectEsmImports(ast, packagesToInspect, deep = true) {
   const shouldInspectAll = !packagesToInspect
   const pkgsToInspect = new Set(packagesToInspect)
 
   /** @type {Set<string>} */
-  const esmImports = new Set()
+  const result = new Set()
 
-  /**
-   * @param {import('@babel/traverse').NodePath<
-   *   | import('@babel/types').ImportDeclaration
-   *   | import('@babel/types').ExportNamedDeclaration
-   * >} path
-   */
-  const handleNodePath = (path) => {
-    const importSource = path.node.source?.value
+  traverse(ast, {
```

Comment:
There are a few other node types that ModuleSource in SES is looking at
https://github.com/endojs/endo/blob/naugtur/e2e-test-conditional-conditions/packages/module-source/src/babel-plugin.js#L313

We don't need all of them because we're not interested in finding exports. I'll try to make a comparison to see if this is not missing anything tho.